modules/distr-tracing-rn-technology-preview.adoc

```diff
@@ -0,0 +1,33 @@
+////
+Module included in the following assemblies:
+* distr-tracing--release-notes.adoc
+////
+
+[id="distr-tracing-rn-tech-preview_{context}"]
+= Technology Preview
+////
+Provide the following info for each issue if possible:
+Description -  Describe the new functionality available to the customer.  For enhancements, try to describe as specifically as possible where the customer will see changes.  Avoid the word “supports” as in [product] now supports [feature] to avoid customer confusion with full support.  Say, for example, “available as a Technology Preview.”
+Package - A brief description of what the customer has to install or enable to use the Technology Preview feature.    (e.g., available in quickstart.zip on customer portal, JDF website, container on registry, enable option, etc.)
+////
+
+[IMPORTANT]
+====
+Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production.
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+== {ProductName} 2.0.0 Technology Preview (OpenShift 4.9)
+
+This release includes the addition of the OpenTelemetry Operator to install distributed tracing data collection.
+
+{ProductName} data collection is based on OpenTelemetry's Operator and Collector. The Collector can be used to receive traces in either the OpenTelemetry or Jaeger protocol and send the trace data to the OpenShift distributed tracing platform. Other capabilities of the Collector are not supported at this time.
+
+== {ProductName} 2.0.0 Technology Preview (OpenShift 4.6 - 4.8)
```

Comment:
I think this is a old version of Jaeger tech preview channel.
![image](https://user-images.githubusercontent.com/1004403/142361120-a8c7bf78-193a-4b4c-9829-84f763007915.png)

We have completely dropped this path
I think it is better to remove from here or update a note

 @pavolloffay @mwringe Please suggest

Reply:
OK, I wasn't sure how this was going to end up looking in the GUI.  Thanks!